SuperBuild.cmake

```
@@ -47,7 +47,7 @@ endif()
 # Find CUDA
 option(DISABLE_CUDA "Disable CUDA" OFF)
 if (NOT DISABLE_CUDA)
-  find_package(CUDA)
+  find_package(CUDAToolkit REQUIRED)
```

```suggestion
  find_package(CUDA)
```
While this is needed, let's do it in another PR, as both of these could create discussions.